package.json

```diff
@@ -9,6 +9,7 @@
     "test-watch": "cross-env NODE_ENV=test karma start --auto-watch --no-single-run",
     "release": "node tasks/release.js",
     "i18n": "cross-env NODE_ENV=i18n webpack && node tasks/i18n",
+    "i18njson": "node tasks/i18njson",
```

Comment:
Where does this fit in with the existing `i18n` task? Should there be two separate tasks?